test/parallel/test-debug-signal-cluster.js

```diff
@@ -23,10 +23,12 @@ var common = require('../common');
 var assert = require('assert');
 var spawn = require('child_process').spawn;
 
-var args = [ common.fixturesDir + '/clustered-server/app.js' ];
-var child = spawn(process.execPath, args, {
-  stdio: [ 'pipe', 'pipe', 'pipe', 'ipc' ]
-});
+var port = common.PORT + 42;
+var args = ['--debug-port=' + port,
+            common.fixturesDir + '/clustered-server/app.js'];
+var options = {stdio: ['inherit', 'inherit', 'pipe', 'ipc']};
```

Comment:
Apropos the style, a quick grep through the test suite finds more occurrences of `[a, b, c]` than of `[ a, b, c ]`.